HW19/user.py

```diff
@@ -0,0 +1,14 @@
+class User:
+    def __init__(self, name):
+        self.name = name.lower()
+
+    def __eq__(self, other):
+        if self.name == other:
```

Comment:
I think it could look like
```suggestion
        return self.name.lower() == other.name.lower():
```